src/mixins/filing-template-mixin.ts

```diff
@@ -427,7 +427,7 @@ export default class FilingTemplateMixin extends DateMixin {
         offices: this.getDefineCompanyStep.officeAddresses,
         parties: this.orgPersonsToParties(this.getAddPeopleAndRoleStep.orgPeople),
         relationships: this.getRestoration.relationships,
-        type: this.getRestoration.type
+        type: this.getRestoration.type || RestorationTypes.FULL
```

Comment:
When saving with restoration type in default state, an error was being shown (unable to save). This is to fix that.